packages/helpers/test/operations.test.js

```diff
@@ -1,17 +1,21 @@
 const path = require('path');
 const { Parser, fromFile } = require('@asyncapi/parser');
-const { getOperationMessages } = require('@asyncapi/generator-helpers');
+const { getOperationMessages, getMessageExamples } = require('@asyncapi/generator-helpers');
 
 const parser = new Parser();
 const asyncapi_v3_path = path.resolve(__dirname, './__fixtures__/asyncapi-websocket-query.yml');
 
+async function getOperationsFromParsedDoc() {
+  const parseResult = await fromFile(parser, asyncapi_v3_path).parse();
+  const parsedAsyncAPIDocument = parseResult.document;
+  return parsedAsyncAPIDocument.operations();
+}
+
 describe('getOperationMessages integration test with AsyncAPI', () => {
-  let parsedAsyncAPIDocument, operations;
+  let operations;
 
   beforeAll(async () => {
-    const parseResult = await fromFile(parser, asyncapi_v3_path).parse();
-    parsedAsyncAPIDocument = parseResult.document;
-    operations = parsedAsyncAPIDocument.operations();
+    (operations = await getOperationsFromParsedDoc());
```

Comment:
all good just these paranthesis are pretty confusing, we do not need them, they do not change anything

```suggestion
    operations = await getOperationsFromParsedDoc();
```